packages/jsonapi/node-tests/middleware-test.js

```diff
@@ -179,7 +179,7 @@ describe('jsonapi/middleware', function() {
       expect(response).hasStatus(200);
       expect(response.body).deep.property('data.id', '0');
       expect(response.body).deep.property('data.attributes.title', 'Hello world');
-      expect(response.body).deep.property('data.relationships.author.data', null);
+      expect(response.body).to.not.have.deep.property('data.relationships');
```

Comment:
note: this was an instance of the seed model being modified as part of validation, the original model never had relationships.